package/MDAnalysis/analysis/pcz.py

```diff
@@ -0,0 +1,481 @@
+"""
+A PCA module for MDAnalysis
+==========================================================================
+
+:Authors: The pyPcazip Developers - Ardita Shkurti, Ramon Goni, Pau Andrio, Elena Breitmoser, Iain
+Bethune, Modesto Orozco, Charles Laughton
+:Year: 2015
+:Copyright: BSD
```

Comment:
Would it be a problem for you if this code could share the same license as the other MDAnalysis code (a GPLv2)?

That would make it easier for us in the long-run to do housekeeping changes on this code.